projects/npm-tools/packages/npm-scripts/src/jest/mocks/Liferay.js

```diff
@@ -134,6 +134,11 @@ const ThemeDisplay = {
  */
 const Util = {
 
+	/**
+	 * https://github.com/liferay/liferay-portal/blob/efc53b8a82e3afff06aacc26118fbe7231acffff/portal-web/docroot/html/common/themes/top_js.jspf#L157
+	 */
+	FeatureFlags: {},
```

Comment:
I think this would be at the root level right and not Util, right? Since it is `Liferay.FeatureFlags` and not `Liferay.Util.FeatureFlags`